woodsman.gemspec

```diff
@@ -19,15 +19,14 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'activesupport', '< 7'
-  spec.add_dependency 'require_all', '~> 1.3'
-  spec.add_dependency 'sentry-raven', '~> 2.3'
+  spec.add_dependency 'sentry-raven', '~> 3.1.2'
 
-  spec.add_development_dependency 'guard-rspec', '~> 4.5'
-  spec.add_development_dependency 'rake', '~> 12.3'
-  spec.add_development_dependency 'rspec', '~> 3.1'
-  spec.add_development_dependency 'rspec_junit_formatter', '~> 0.2'
-  spec.add_development_dependency 'metric_fu', '~> 4.11'
-  spec.add_development_dependency 'simplecov', '~> 0.9'
-  spec.add_development_dependency 'webmock', '~> 1.18'
-  spec.add_development_dependency 'timecop', '~> 0.7'
+  spec.add_development_dependency 'guard-rspec', '~> 4.7'
```

Comment:
Since we're changing the dependencies, we should also bump the version on the woodsman gem.

Reply:
version.rb

Reply:
Updated!